### PR TITLE
Announce oasis releases

### PR DESCRIPTION
**Pull request by paytonrules**
Add release tracking.

### CODE REVIEW

File: .travis/deploy.sh [hunk 1]
```diff
@@ -38,4 +38,8 @@ if [ "$ENVIRONMENT" == "prod" ]; then
     aws ecs update-service --cluster vdb-cluster-$ENVIRONMENT --service vdb-oasis-execute-$ENVIRONMENT --force-new-deployment --endpoint https://ecs.$PROD_REGION.amazonaws.com --region $PROD_REGION
 else
     message UNKNOWN ENVIRONMENT
+    exit 1
```
Comment (rmulhol) [inline .travis/deploy.sh:41]: Maybe this belongs right after line 17?

File: transformers/announce.sh [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+else
+   message UNKNOWN ENVIRONMENT
+   echo 'Allowed values for environment are "staging" or "prod"'
+   exit 1
+fi
+
+PROCESS=$2
+if [ -z "$PROCESS" ]; then
+    message NO PROCESS NAME
+    echo 'You must specify process name, such as vdb-execute (bash announce.sh <ENVIRONMENT> <PROCESS>).'
```
Comment (rmulhol) [inline transformers/announce.sh:31]: nbd but maybe the example could be `vdb-oasis-execute` instead of `vdb-execute`

**Review by rmulhol — APPROVED** [review]

File: transformers/announce.sh (hunk 2 above)
Comment (paytonrules) [reply transformers/announce.sh:31]: Fixed - but more importantly this file is in the wrong place!

File: .travis/deploy.sh (hunk 1 above)
Comment (paytonrules) [reply .travis/deploy.sh:41]: 👍

**Pull request merged**